1-exercises/C-undefined-properties/exercise.js

```diff
@@ -25,14 +27,17 @@ let user = {
   name: "Mira"
 };
 
-sayHelloToUser(user);
+sayHelloToUser(user); 
+//There's no property 'firstName' in the 'user' object
 
 // Example 3
 let myPet = {
   animal: "Cat",
   getName: function() {
-    "My pet's name is Fluffy";
+     "My pet's name is Fluffy";
   },
 };
 
-console.log(myPet.getName());
+console.log(myPet.getName()); 
+//There's no return or log value from myPet object's getName() function, so it will return 'undefined'.
```

Comment:
It is true that the 'undefined' is here because there is no return. Even if there was a log value in the getName, it would still be undefined.